public/romania/manifest.json

```
@@ -1,72 +1,72 @@
 {
   "name": "BBC News Romania",
   "short_name": "BBC Romania",
-  "scope": "/romania",
+  "scope": "/Romania",
   "background_color": "#b80000",
   "display": "standalone",
   "theme_color": "#b80000",
   "icons": [
     {
-      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/magyarul/images/icons/icon-72x72.png?v=2",
+      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/afaanoromoo/images/icons/icon-72x72.png?v=1",
```

If it doesn't cause any issues, let's do this for all icons
```suggestion
      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/romania/images/icons/icon-72x72.png?v=1",
```